quantities.hxx

```diff
@@ -57,8 +57,92 @@ auto phi(auto df, const std::string &outputname,
                      },
                      {inputvector});
 }
-/// Function to calculate the mass from a pair of lorentz vectors and add it to
-/// the dataframe
+/// Function to writeout the mass from a particle. The particle is identified
```

Comment:
The thing is, that the function has to be different depending on the datatype of the output, e.g. charge is an int, genmatch is a uint etc. This would mean, that we need several of those generic functions, and as soon as something more complicated has to be done, these basic functions wont work anymore. So i would suggest to keep it this way, with one function per quantity, even if this results in some duplications, for consistency reasons.